server/logger.js

```diff
@@ -47,9 +47,9 @@ const winstonInstance = new winston.Logger({
  */
 const loggerConf = {
     winstonInstance: winstonInstance,
-    dynamicMeta: getMeta,
```

Comment:
Why you removed dynamic meta?
How will it be possible to see who is executing requests in the logs?

Reply:
@a1exsh

Reply:
I was outraged that it also mucks with the access token.  I guess I've missed the part where it adds the user.  I will put it back, but omit the token hash.